utils/manage_events.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+
+import logging
```

Comment:
I saw this file in @SwapnilGaonkar7's patch. Why do I see it appearing in your diff?

Reply:
Oh, Actually it was decided to merge this to main. So I changed it main, but it was rebased with the fault_tolerance branch before.

Reply:
Yes rebased with main and dropped all commits of fault tolerance